move/interchain_token_service/sources/types/token_manager_types.move

```diff
@@ -6,10 +6,15 @@ module interchain_token_service::token_manager_types {
         MINT_BURN, // The token will be minted/burned on transfers. The token needs to give mint and burn permission to the token manager.
```

Comment:
move TokenManagerType to ITSv0 so you can use the variant directly and match on it